client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -98,19 +94,21 @@ module.exports = {
 			'Domain Management › Set Primary Domain'
 		);
 
-		renderPage(
+		renderWithReduxStore(
 			<PrimaryDomainData
 				component={ DomainManagement.PrimaryDomain }
-				context={ context }
-				selectedDomainName={ context.params.domain }
-				sites={ sites } />
-		);
+				pageContext={ pageContext }
```

Comment:
It looks like `pageContext` is never used for this Data component: https://github.com/Automattic/wp-calypso/blob/master/client/components/data/domain-management/primary-domain/index.jsx.